### DISCUSSION TITLE
Special Paste like Ditto

### DISCUSSION THREAD

**Discussion by kayanco** [hluk/CopyQ#2358] [Ideas]
Any chance of adding a "Special Paste" menu item like in Ditto:
https://i.imgur.com/sM6uqdC.jpeg

I think it would be helpful. Thoughts on this please?

**Comment by greg4b** [comment]
I'd like especially the option 

- MULTIPLE IMAGES HORIZONTALLY 
- MULTIPLE IMAGES VERTICALLY 

Thanks,

Gregor

**Comment by kayanco** [comment]
Bumping this request, if possible

**Comment by hluk** [comment]
This custom functionality can be added via commands. For example, here is [Capitalize Selected Text for Titles](https://github.com/hluk/copyq-commands/blob/master/Global/README.md#capitalize-selected-text-for-titles) (see [README.md](https://github.com/hluk/copyq-commands/blob/master/README.md) for details on how to add the command).
